versioned/spi/src/main/java/org/projectnessie/versioned/VersionStore.java

```diff
@@ -150,12 +156,17 @@ void transplant(
    * @param fromHash The hash we are using to get additional commits
    * @param toBranch The branch that we are merging into
    * @param expectedHash The current head of the branch to validate before updating (optional).
+   * @param updateCommitMetadata Function to allow resetting properties application due to merge.
```

Comment:
Please reword. "resetting properties application" is confusing.

Reply:
ok